drafts/Stormless.md

```diff
@@ -0,0 +1,198 @@
+TODO: Add the beginning of the story. Put up excuses and disclamers.
```

Comment:
I'm assuming you're planning to put this here before publishing still - just want to emphasize that this is definitely needed. Especially if this is first in a series, give the reader a sense of what they can expect to get out of each episode, and then the series as a whole. Since this is the first episode, they'll want to get a reason early on to keep reading.

I highly recommend you tie this in to https://stackstorm.com/2017/12/14/stackstorm-exchange-goes-serverless/, with this blog linking to that, and vice versa.